docs/changelog.md

```diff
@@ -1,3 +1,5 @@
+* 0.2.0-alpha
```

Comment:
```suggestion
* HEAD
```

Let's not name the release until it actually goes out. Multiple branch may be in progress in parallel and some may not make the cut for a particular release.

Reply:
Oh, Yes. That makes sense